README.md

```diff
@@ -0,0 +1 @@
+Hello world!
```

Comment:
Should we use a `.` instead of `!`?

Reply:
Yep.

Reply:
@awflwafl Is this another one of your accounts @elixier01 ?

Reply:
🙂@BarryGallagher That's Andrew.